samples/02_power_users_developers/introduction_to_hub_initiatives_and_sites.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>In the interest of demonstrating the Hub Premium <code>Initiative</code>, we will sign into a Hub. Sign-in with your ArcGIS credentials while executing this sample notebook to leverage the capabilities of your Hub/Sites licensing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>